src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -1265,208 +1488,226 @@ instruct replicateD(vReg dst, fRegD src) %{
 
 // vector shift
 
-instruct vasrB(vReg dst, vReg src, vReg shift) %{
+instruct vasrBS(vReg dst, vReg src, vReg shift, vRegMask_V0 tmp) %{
```

Comment:
I really don't like instruct names like 'vasrBS' and 'vasrIL' which looks kind of misleading. I would perfer keep those seperated like before.

Reply:
Fixed.